go.mod

```diff
@@ -23,7 +23,7 @@ require (
 	github.com/aristanetworks/glog v0.0.0-20191112221043-67e8567f59f3 // indirect
```

Comment:
do we need to bump the go version as well?

Reply:
No immediate rush, but def need to do this eventually. There's upstream changes to pull down too, but I have been postponing these maintenance tasts until we do some more design/planning. I'll create some issues for tracking at least.